code/modules/projectiles/ammunition/magazines.dm

```diff
@@ -258,6 +258,51 @@
 	caliber = "4.6x30mm"
 	max_ammo = 20
 	multi_sprite_step = 4
+	multiload = 0
+	w_class = WEIGHT_CLASS_NORMAL
+	///A var to check if the mag is being loaded
+	var/being_loaded = FALSE
+	/// There are two reloading processes ongoing so cancel them
+	var/double_loaded = FALSE
+
+/obj/item/ammo_box/magazine/wt550m9/attackby(obj/item/A, mob/user, params)
+	if(istype(A, /obj/item/ammo_box/wt550) || istype(A, /obj/item/ammo_box/magazine/wt550m9))
+		to_chat(user, "<span class='notice'>You begin to load the magazine with [src].</span>")
```

Comment:
```suggestion
		to_chat(user, "<span class='notice'>You begin to load the magazine with [A].</span>")
```
I think you intended for this?

Reply:
should have the same result no? Message displayed properly when I tested it iirc

Reply:
No, src here would print the message that you're loading the ammo box with the ammo box, not the ammo.

Reply:
Yep you were right, never noticed that in testing. Applied the change in last commit